sys/src/cmd/frontusb/cam/cam.c

```diff
@@ -232,7 +232,7 @@ fsread(Req *req)
 	}
 	f = req->fid->file;
 	c = f->aux;
-	if(req->fid->aux == nil || req->ifcall.offset == 0)
+	if(req->fid->aux == nil || req->ifcall.offset == 0) {
```

Comment:
wow, this is a nice fix by spatch. That existing code was an accident waiting to happen.

Reply:
It wasn't spatch - it was found by gcc and fixed by hand.